app/views/collection_drafts/forms/fields/_geometry_fields.html.erb

```diff
@@ -32,8 +32,8 @@
       prefix: name_prefix,
       required: true
     ) %>
-    <a href="javascript:void(0)" class="clear-radio-button" data-content="coordinate-system-type">Clear</a>
-    <div class="coordinate-system-type-group">
+    <a href="javascript:void(0)" class="clear-radio-button" data-clear-fields="coordinate-system-type-clear-link-clear-fields" data-hide-fields="coordinate-system-type-clear-link-hide-fields" data-radio-fields="coordinate-system-type-clear-link-radio-fields">Clear</a>
```

Comment:
should this say: resolution-and-coordinate-system-type to be consistent?

Reply:
So this is the reason I changed the other radio buttons to `resolution-and-coordinate-system-type`. This is actually correct and for a different field. So the other was kind of a dupe and made for confusion.